Cargo.toml

```diff
@@ -1,86 +1,34 @@
-[package]
+[workspace]
+resolver = "2"
+
+members = [
+        "examples",
+        "micro-rdk",
+        "micro-rdk-installer",
+        "micro-rdk-macros",
+]
+
+[workspace.package]
 authors = ["Viam, inc. <contact@viam.com>"]
 description = "Viam RDK for microcontroller"
 edition = "2021"
 license = "AGPL-3.0"
-name = "micro-rdk"
 repository = "https://github.com/viamrobotics/micro-rdk"
-resolver = "2"
-version = "0.1.6"
 rust-version = "1.71"
-links = "micro_rdk"
-
-[lib]
-crate-type = ["lib"]
+version = "0.1.6"
 
 [patch.crates-io]
```

Comment:
I think it make sense for now to keep the patches here for now since examples & micro-rdk needs them when building thier respective targets.